appium-tests/helpers/cameraHelper.js

```diff
@@ -1,311 +0,0 @@
-/* global Q, resolveLocalFileSystemURL, Camera, cordova */
```

Comment:
file removed?

Reply:
I was wondering the same. CI is green though.

Reply:
Appium tests are currently removed from the project, as paramedic can't properly run them. PR https://github.com/apache/cordova-plugin-camera/pull/469 adds them back when the problems are fixed. But I am confused why this PR removes that file - because it should already be removed.

Reply:
This is actually weird because this file was removed from master as a certain point in time. I will re-sync with master again and resolve any merge conflicts.

Reply:
Checking it out, I think appium-tests/helpers/cameraHelper.js is removed in this commit:
https://github.com/apache/cordova-plugin-camera/commit/19d8e2f6ff230c1de51f247cec30b71134971507

Reply:
My PR had some updates to this `cameraHelper.js` file in order to assert that  DATA_URL is properly prefixed, so not having this unit test can cause issues while trying to fix https://github.com/apache/cordova-plugin-camera/pull/469. So here is my proposed plan:
1. Postponing my PR #432 until #469 is resolved.
2. Once PR #469 is resolved, I will sync-up and add the relevant unit tests for the DATA_URL feature.
3. Once finished,  PR #432 will be re-pushed.
Does this sound a good plan?